doc/boxes.el

```diff
@@ -35,76 +35,72 @@
 ;;          (global-set-key "\C-cr" 'boxes-remove)
 
 ;;; Code:
-(eval-when-compile (require 'cl))
 
-;;;###autoload
-(defvar boxes-command "boxes"
-  "The boxes command.")
+(defgroup boxes nil
+  "ASCII boxes unlimited!"
+  :group 'convenience)
 
-(defvar boxes-types-alist
-  (let ((the-types (process-lines boxes-command "-q" "(all)"))
-        (the-alist (list)))
-    (dolist (el the-types the-alist)
-      (let ((no-alias (replace-regexp-in-string " *\(alias\) *$" "" el)))
-        (add-to-list 'the-alist 
-                     (cons no-alias no-alias)))))
-  "Association of types available to the current boxes implementation." )
-(make-variable-buffer-local 'boxes-types-alist)
+(defcustom boxes-command "boxes"
+  "The boxes command."
+  :type 'string
+  :group 'boxes)
 
-(defvar boxes-history nil)
+(defcustom boxes-args ""
+  "Additional arguments to the boxes command."
+  :type 'string
+  :group 'boxes)
```

Comment:
Made ‘boxes-command’ customisable.

Re-ordered the variable declarations to avoid some compiler warnings